pypeit/tests/test_scripts.py

```diff
@@ -195,12 +196,8 @@ def test_coadd1d_1():
     if os.path.isfile(coadd_ofile):
         os.remove(coadd_ofile)
 
-    # TODO: This is a kludge to get Travis to work.  We need a script
-    # that will write the coadd files with absolute paths.
-    coadd_ifile = data_path('shane_kast_blue.coadd1d') if os.getenv('TRAVIS_BUILD_DIR') is None \
-                    else data_path('shane_kast_blue_travis.coadd1d')
-
-    coadd_1dspec.main(coadd_1dspec.parser([coadd_ifile]))
+    coadd_ifile = data_path('shane_kast_blue.coadd1d')
+    coadd_1dspec.main(coadd_1dspec.parser([coadd_ifile]), test_spec_path=data_path(''))
```

Comment:
Ah, I see.  Can we instead make the root path an argument to the coadd_1dspec script?  I'm sure I'm being overly pedantic, but I worry about adding specific behavior only for the tests.

Reply:
ok, but I've kept the name `test_spec_path` as I don't really want a user to use it.